lib/google_drive/session.rb

```diff
@@ -418,18 +418,39 @@ def collection_by_url(url)
 
     alias folder_by_url collection_by_url
 
-    # Creates new spreadsheet and returns the new GoogleDrive::Spreadsheet.
+    # Creates a spreadsheet with given title. Returns GoogleDrive::Spreadsheet
```

Comment:
Missing methods: `create_subcollection` and `alias create_subfolder create_subcollection`